### PR TITLE
aws-s3-rm: add page

### PR DESCRIPTION
**Pull request by shermanhui**
This PR addresses #8406 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Screen Shot 2022-10-20 at 19 48 20

### CODE REVIEW

File: pages/common/aws-s3-rm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# aws s3 rm
+
+> Delete an S3 object.
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:3]:
```suggestion
> Delete S3 objects.
```

File: pages/common/aws-s3-rm.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/rm.html>.
+
+- Delete a single S3 object:
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:6]:
```suggestion
- Delete a specific S3 object:
```

File: pages/common/aws-s3-rm.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:8]:
```suggestion
`aws s3 rm s3://{{bucket_name}}/{{path/to/file}}`
```

File: pages/common/aws-s3-rm.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Perform a dry-run of a single S3 object deletion:
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:10]:
```suggestion
- Preview the deletion of a specific S3 object without delete it (dry-run):
```

File: pages/common/aws-s3-rm.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt --dryrun`
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:12]:
```suggestion
`aws s3 rm s3://{{bucket_name}}/{{path/to/file}} --dryrun`
```

File: pages/common/aws-s3-rm.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Delete an object from an S3 access point:
+
+`aws s3 rm s3://arn:aws:s3:us-west-2:123456789012:{{access_point}}/{{my_access_point}}/{{my_key}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:16]: I think this is the correct placeholders
```suggestion
`aws s3 rm s3://arn:aws:s3:{{region}}:123456789012:access_point/{{access_point_name}}/{{object_key}}`
```

File: pages/common/aws-s3-rm.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Delete a single S3 object:
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt`
+
+- Perform a dry-run of a single S3 object deletion:
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt --dryrun`
+
+- Delete an object from an S3 access point:
```
Comment (navarroaxel) [inline pages/common/aws-s3-rm.md:14]:
```suggestion
- Delete an object from a specific S3 access point:
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/aws-s3-rm.md (hunk 6 above)
Comment (shermanhui) [reply pages/common/aws-s3-rm.md]: @navarroaxel I've updated it to look like this example from the AWS user guide [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html)

```
arn:aws:s3:region:account-id:accesspoint/access-point-name/object/resource
```

Let me know if there shouldn't be curly braces around the `account_id` and `access_point` values!

Thanks for pointing this out

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**